### PR TITLE
Update manifest as Vapor is no longer in RC 🎉

### PR DESCRIPTION
**Pull request by 0xTim**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -8,7 +8,7 @@ let package = Package(
     ],
     dependencies: [
         // 💧 A server-side Swift web framework.
-        .package(url: "https://github.com/vapor/vapor.git", from: "4.0.0-rc"){{#fluent}},
+        .package(url: "https://github.com/vapor/vapor.git", from: "4.5.1"){{#fluent}},
```
Comment (tanner0101) [inline Package.swift:11]: Putting such a specific version here seems kind of weird to me. Theoretically everything in this template _is_ compatible with version 4.0.0, so why wouldn't we put that? It seems like if we put something more specific we should be sure it is updated regularly. 

I may be way overthinking this though >_<. What does everyone else think?

**Comment by tanner0101** [comment]
> Please wait for @tanner0101 's review, I believe he wanted to wait on this.

If I did I can't remember why >_>. Hopefully it wasn't important.

File: Package.swift (hunk 1 above)
Comment (MrLotU) [reply Package.swift:11]: Would it be overkill (probably a yes) to make it a template variable and just pick up the latest released version whenever someone initialises a new repo? Would only be a simple GH API Req

Comment (gwynne) [reply Package.swift:11]: Seems a bit excessive given that it's what `from` and `upToNextMajor` and `upToNextMinor` are there for in SPM to begin with 🤷‍♀️

Comment (MrLotU) [reply Package.swift:11]: Agreed. In that case I'd just put down `4.0.0` :)

Comment (0xTim) [reply Package.swift]: @gwynne updated!

**Pull request merged**